std_msgs/Header header
detected_obstacle[] obstacles
